package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "typed-om",
```

Comment:
For JS files I agree, but in a JSON file that’s strictly speaking invalid as the spec demands double quotes and `npm` actually fails parsing with single quotes.